src/Search/Search.Management.Tests/app.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```

Comment:
you can get rid of this file as well, unless there is common nuget scenario which causes this file to get added back automatically, which i understand :-)